src/index.js

```diff
@@ -4,40 +4,72 @@ import filter from 'feathers-query-filters';
 import errors from 'feathers-errors';
 import { select } from 'feathers-commons';
 import * as utils from './utils';
+import { Op } from 'sequelize';
 
 class Service {
   constructor (options) {
     if (!options) {
       throw new Error('Sequelize options have to be provided');
     }
 
-    if (!options.Model) {
-      throw new Error('You must provide a Sequelize Model');
+    if (!options.model) {
```

Comment:
We use a model name here (instead of a `Model` instance), since we need to retrieve the model from each `Sequelize` instance. To retain support for setups without replication, this must no longer be mandatory. We could allow providing either an instance or a name, but not both. I used `model` here (with a lowercase m), but it's probably going to be confusing if we support both `Model` and `model`.

Reply:
Due to the existing API as well as the API's for other adapters, it would be super nice to keep `Model` as the option. We can get the "name" from the Model constructor and use that internally. This will also keep this module backwards compatible.